.github/workflows/release.yml

```
@@ -3,8 +3,30 @@ on:
   push:
     tags:
       - 'v*'
+
 jobs:
+  test:
+    name: run tests before release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.9)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)